conf/arm64.src

```diff
@@ -3,4 +3,4 @@ SOURCE_SUM=5e98df2198e7dfe6479d565e82fd3e68a07d2334435a05c327e69de4264d54f7
 SOURCE_SUM_PRG=sha256sum
 SOURCE_FORMAT=zip
 SOURCE_IN_SUBDIR=false
-#SOURCE_FILENAME=
+SOURCE_FILENAME=mautrix-whatsapp
```

Comment:
As is `amd64.src`
should we add `SOURCE_EXTRACT=false`
and remove `SOURCE_FORMAT=zip`
or is it explicit?